group/assignment6/Assignment6.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#include <vector>
+
+int findVacantSlot(const std::vector<int> &parking_lot) {
+	for (int i = 0; i < parking_lot.size();i++) {
+		if (parking_lot[i] == -1)
+			return i;
+	}
+}
+std::vector<int> getCarsWrongPlace(const std::vector<int> &starting_state, const std::vector<int> &ending_state) {
+	std::vector<int> cars_wrong_place;
+	for (int i = 0; i < starting_state.size(); i++) {
+		if (starting_state[i] != ending_state[i] && starting_state[i] != -1)
+			cars_wrong_place.push_back(starting_state[i]);
```

Comment:
You could make a variable for starting_state[i], for example, current_slot - to reuse it 3 times